hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/om-overview.html

```diff
@@ -22,6 +22,18 @@ <h2>Status</h2>
         <td>Rpc port</td>
         <td>{{$ctrl.overview.jmx.RpcPort}}</td>
     </tr>
+    <tr>
+        <td>OM Roles (HA)	</td>
+        <td>{{$ctrl.overview.jmx.OmRatisRoles}}</td>
+    </tr>
+    <tr>
+        <td>Current-Host</td>
```

Comment:
Can  remove current host field

Reply:
I have removed it and replaced it with current role 
Thanks !!